style.scss

```diff
@@ -30,6 +30,24 @@ Underscores is distributed under the terms of the GNU GPL v2 or later.
   --transition-duration: 333ms;
 }
 
+figcaption a{color: #FFF!important;}
```

Comment:
Should this be inside the media query?

Reply:
Yes, I think it should've been in the media query, don't know how I messed this up.
Probably we should set the line-height to something like 1.9.